### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kengying**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -10,17 +12,14 @@ ifndef::env-github[]
...
-** A more sophisticated GUI that includes a list  panel and an in-built Map.
-** The ability to store contacts online and retrieve them on any device.
-** A better system to group the user's contacts with multiple Address Books, Groups and tags.
-** More fluid and powerful CLI Commands.
-** Export contacts to a file.
-** Share Contacts to another user's Address Books.
-** Power users can enable CLI through Preferences.
+Too many clients on hand? Not enough time to save their details?
+
+Welcome to *humanDEX*! This is an addressbook specially made for all insurance agent!
```
Comment (sofarsophie) [inline README.adoc]: all insurance agents?

File: README.adoc [hunk 2]
```diff
@@ -10,17 +12,14 @@ ifndef::env-github[]
...
-* It is *written in OOP fashion*.
-* What's different from https://github.com/se-edu/addressbook-level4[other address books]:
-** A more sophisticated GUI that includes a list  panel and an in-built Map.
-** The ability to store contacts online and retrieve them on any device.
-** A better system to group the user's contacts with multiple Address Books, Groups and tags.
-** More fluid and powerful CLI Commands.
-** Export contacts to a file.
-** Share Contacts to another user's Address Books.
-** Power users can enable CLI through Preferences.
+Too many clients on hand? Not enough time to save their details?
```
Comment (sofarsophie) [inline README.adoc]: i like this part :D

File: README.adoc (hunk 1 above)
Comment (kengying) [reply README.adoc]: uh, i'll take the all away

Comment (kengying) [reply README.adoc]: but it will sound weird tho. any suggestions?

**Pull request merged**